Add new procedure for errata card (#1976)
theforeman/foreman-documentation: guides/common/modules/proc_viewing-errata-by-applicable-and-installable.adoc

[id="Viewing_Errata_by_Applicable_and_Installable_{context}"]
= Viewing Errata by Applicable and Installable

Use the following procedure to view errata by Applicable or Installable.

.Procedure
. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
. Select the host name.

> . Click the *Overview* tab. Under the Errata card, there are two types of Errata.

Reviewer: ```suggestion
. Click the *Overview* tab.
Under the Errata card, there are two types of Errata.
```

> . Click *Applicable* to view the type of Errata that applies to a package installed on your host.

Reviewer: ```suggestion
. Click *Applicable* to view errata that apply to a package installed on your host.
```

> . Click *Installable* to view errata that are applicable errata available in the host’s content view and lifecycle environment.

Reviewer: Aren't these supposed to be the opposite?

Author: Should it be reworded? @adamlazik1

Reviewer: Could you please double check this with SME? To me, the description would make more sense if it was reversed. Could also be a misunderstanding on my part but still I will be happier if we can have this confirmed.

Author: @adamlazik1 Spoke to the SME and he said what I have is correct. Installable is a subset of Applicable.

Reviewer: :+1:

> . Click the available number link under each type to see the list of all available errata.

Reviewer: ```suggestion
. Click the link with number of errata under each type to see the list of all available errata of that type.
```

> . Click the other links for each type to filter *security advisories*, *bug fixes*, and *enhancements*.

Reviewer: ```suggestion
. Click *security advisories*, *bug fixes*, or *enhancements* under each type to view only the respective type of errata.
```

[id="Viewing_Errata_by_Applicable_and_Installable_{context}"]
= Viewing Errata by Applicable and Installable

> Use the following procedure to view errata by Applicable or Installable.

Reviewer: ```suggestion
Use the following procedure to view errata by applicable or installable.
```
